concert/experiments/imaging.py

```diff
@@ -127,6 +127,10 @@ def __init__(self, walker, flat_motor, radio_position, flat_position, camera, nu
         super().__init__([darks_acq, flats_acq, radios_acq], walker,
                          separate_scans=separate_scans)
 
+    async def prepare(self):
+        if await self._camera.get_state() != "standby":
+            await self._camera.stop_recording()
```

Comment:
So useful... We should maybe check all the device states here (but that's a separate story).